shared/index.cjsx

```diff
@@ -9,6 +10,9 @@ module.exports = {
   OXRouter:               require './src/helpers/router'
   OXLink:                 require './src/factories/link'
   OXMatchByRouter:        require './src/helpers/match-by-router'
+  OXMatchByRouter:        require './src/helpers/match-by-router'
+  APIHandler,
+  APIActionAdapter:       require './src/api/action-adapter'
 
   ArbitraryHtmlAndMath:   require './src/components/html'
   Markdown:               require './src/components/markdown'
```

Comment:
hmmm i agree it's getting ridiculous, but i want to ponder this decision a little more.  i dont exactly like the idea of having to look up the location of all the files either and the long paths in all the places we're using `shared` stuffs.  i kinda like that accessing the things we want from shared is very easy, but this is getting a little ridiculous.